code/lib/cli/src/js-package-manager/JsPackageManager.ts

```diff
@@ -134,6 +134,8 @@ export abstract class JsPackageManager {
 
     done = commandLog('Installing dependencies');
 
+    logger.log();
```

Comment:
this adds a newline so that the package manager's install logs are not written to the same line